modules/@apostrophecms/schema/index.js

```diff
@@ -1090,7 +1090,7 @@ module.exports = {
           const idsStorage = field.idsStorage;
           const ids = await query.toDistinct(idsStorage);
           const manager = self.apos.doc.getManager(field.withType);
-          const relationshipQuery = manager.find(query.req, { aposDocId: { $in: ids } }).project(manager.getAutocompleteProjection({ field: field }));
+          const relationshipQuery = manager.find(query.req, { aposDocId: { $in: ids } }).project(manager.getRelationshipQueryBuilderChoicesProjection({ field: field }));
```

Comment:
I think this is a step too far. Existing projects may be overriding getAutocompleteProjection, because it is used here. So I think we can't stop calling it altogether. We can call the new method, which can be documented as the preferred name, but the new method has to call `getAutocompleteProjection` so that existing overrides don't break. Unfortunately.